uaclient/clouds/identity.py

```diff
@@ -11,13 +13,32 @@
     # typing isn't available on trusty, so ignore its absence
     pass
 
+# Needed for Trusty-only
 CLOUDINIT_RESULT_FILE = "/var/lib/cloud/data/result.json"
+CLOUDINIT_INSTANCE_ID_FILE = "/var/lib/cloud/data/instance-id"
 
 
 # Mapping of datasource names to cloud-id responses. Trusty compat with Xenial+
 DATASOURCE_TO_CLOUD_ID = {"azurenet": "azure", "ec2": "aws"}
 
 
+def get_instance_id(
+    iid_file: str = CLOUDINIT_INSTANCE_ID_FILE
```

Comment:
This is only used in testing, so perhaps:

```suggestion
    _iid_file: str = CLOUDINIT_INSTANCE_ID_FILE
```

Reply:
thanks grabbed it